### DISCUSSION TITLE
WebDav support for jianguoyun

### DISCUSSION THREAD

**Discussion by trganda** [TriliumNext/Trilium#3167] [Ideas] [answered]
I'm trying to write a `js backend` script to upload the local data directory to [jianguo](https://www.jianguoyun.com/) with webdav. It can be got from [https://github.com/trganda/trillium-webdav](https://github.com/trganda/trillium-webdav).

But it looks like not a good manner to do.

It's there any plan to support webdav protocol in trilium?

**Comment by zadam** [comment, marked as answer]
> It's there any plan to support webdav protocol in trilium?

No, not really.
